src/bin/bench.rs

```diff
@@ -0,0 +1,489 @@
+extern crate clap;
+extern crate num_cpus;
+extern crate rsdb;
+#[macro_use]
+extern crate log;
+
+use std::error::Error;
+use std::io::prelude::*;
+use std::process;
+use std::collections::HashMap;
+use clap::{App, Arg};
```

Comment:
this is sort of pedantic, but I usually like to separate the standard library and external imports into different blocks, to help me see off the bat which things are external